ArduPlane/quadplane.cpp

```diff
@@ -1815,7 +1819,9 @@ void QuadPlane::control_run(void)
     switch (plane.control_mode) {
     case QACRO:
         control_qacro();
-        break;
+        // QACRO uses only the multicopter controller
+        // so skip the Plane attitude control calls below
+        return;
```

Comment:
We still need to run the plane controller for none tailsitters

Reply:
In QACRO, the plane controller would be fighting the MC controller on the control surface ouputs. Are there other things the plane controller needs to do during QACRO? I don't see how you would be able to use the forward motors with QACRO in a non-tailsitter... is that what you're thinking about?

Reply:
on standard quadplanes the copter controller does the motors and asks the plane controller to also match the desired rates using the control surfaces. So by skipping the update you loss control surfaces. Tailsitters are a special case because the copter controller also contolls the control surfaces.

Reply:
OK, but do you agree that we need to skip those calls when the vehicle is a tailsitter?
BTW, I haven't done any testing of QACRO on non-tailsitters and regard making it work for those vehicles as a separate task.

Reply:
we don't need to skip, the outputs get overwritten in the tailsitter bit anyway. You could avoid running some unnecessary code by skipping I guess.